pkgs/development/python-modules/langchain-classic/default.nix

```diff
@@ -104,7 +104,10 @@ buildPythonPackage rec {
   ];
 
   # Bulk updater selects wrong tag (there is no tag for this yet)
-  passthru.skipBulkUpdate = true;
+  passthru = {
+    skipBulkUpdate = true;
+    updateScript = false;
```

Comment:
interesting, we have many instances of skipBulkUpdate, but this ( as far I can tell) first instance of `updateScript = false`

not for this PR, but maybe we should merge two flags